config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/PrivateKey.java

```diff
@@ -122,4 +129,38 @@ public PrivateKey build() {
             return new PrivateKey(this);
         }
     }
+
+    @Override
+    public void postLoad(String entityKey, Bundle bundle, File rootFolder, IdGenerator idGenerator) {
+        if (rootFolder != null) {
+            loadPrivateKey(this, new File(rootFolder, "config/privateKeys"), false);
+        }
+
+        setAlias(entityKey);
+        setKeyStoreType(KeyStoreType.fromName(getKeystore()));
+    }
+
+    public static void loadFromDirectory(Collection<PrivateKey> privateKeys, File privateKeysDirectory, boolean failOnMissingKeys) {
+        if (!privateKeys.isEmpty()) {
+            if (!privateKeysDirectory.exists() && failOnMissingKeys) {
+                throw new ConfigLoadException("Directory specified for private keys does not exist: " + privateKeysDirectory.getPath());
+            }
+
+            // load p12 file
+            if (privateKeysDirectory.exists()) {
+                privateKeys.forEach(k -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove useless curly braces around statement ([squid:S1602](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1602))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=128&issues=AWcPsqr_d6oxh3R491Gx&open=AWcPsqr_d6oxh3R491Gx)

[ISSUE_KEY]: AWcPsqr_d6oxh3R491Gx